ironfish/src/wallet/wallet.ts

```diff
@@ -1308,21 +1308,45 @@ export class Wallet {
       throw new Error(`Account already exists with the name ${toImport.name}`)
     }
 
```

Comment:
I wonder if it is worth it to have two separate import methods for the two types?

Reply:
something like:
```
  async importAccount(toImport: AccountImport): Promise<Account> {
    if (toImport.name && this.getAccountByName(toImport.name)) {
      throw new Error(`Account already exists with the name ${toImport.name}`)
    }
    if (spendingKey in toImport) {
       return importSpendingAccount(toImport)
    }  else {
       return importViewOnlyAccount(toImport)
    }
}
```
? We could do that. When the two paths were only slightly different, keeping it as one method made more sense, but now it's an awful lot of code all in one block. Implementing.